docs/multisigs/alliance.md

```diff
@@ -0,0 +1,97 @@
+# Alliance
+
+## 2.1. Lido Alliance BORG Operational
```

Comment:
Why it is 2.1? It should be 3.1 as I understand and all the other pages and poits must be updated